packages/relay/src/routes/user.ts

```diff
@@ -0,0 +1,53 @@
+import { Express } from 'express'
+import { DB } from 'anondb/node'
+import { Synchronizer } from '@unirep/core'
+import TwitterClient from '../singletons/TwitterClient'
+import crypto from 'crypto';
+
+const STATE = crypto.randomUUID()
```

Comment:
what's the purpose of this?

Reply:
Refer to https://developer.twitter.com/en/docs/authentication/oauth-2-0/authorization-code 
To avoid CSRF attack, makes sure the login request is coming from your server